diff --git a/server/techtools.c b/server/techtools.c
index a78d301..7ba51d6 100644
--- a/server/techtools.c
+++ b/server/techtools.c
@@ -677,7 +677,7 @@ pick_random_tech_to_lose(const struct research *presearch)
 
   advance_index_iterate(A_FIRST, i) {
     if (research_invention_state(presearch, i) != TECH_KNOWN) {
-      if (BV_ISSET(eligible_techs, i)) {
+      if (BV_ISSET(eligible_techs, i - 1)) {
         eligible--;
         BV_CLR(eligible_techs, i);
       }
